install_config/persistent_storage/persistent_storage_nfs.adoc

```diff
@@ -168,56 +321,89 @@ To enable writing to NFS volumes with SELinux enforcing on each node, run:
 # setsebool -P virt_use_nfs 1
```

Comment:
@pmorie @jeffvance `virt_sandbox_use_nfs` came up a few months back in https://github.com/openshift/openshift-docs/pull/1250#issuecomment-160999380. Do both need to be set here, and similarly both of the `fusefs` ones in the GlusterFS topic?